### ISSUE TITLE
Adapter requires C++14

### ISSUE THREAD

**Issue by davidscn** [precice/openfoam-adapter#263]
```bash
FSI/Force.C:22:29: error: 'make_unique' is not a member of 'std'
   22 |         ForceOwning_ = std::make_unique<volVectorField>(volVectorField(
      |                             ^~~~~~~~~~~
FSI/Force.C:22:29: note: 'std::make_unique' is only available from C++14 onwards
```

I just realized that the c++ standard depends on the OpenFOAM build. On the cluster (e.g. SuperMUC) using v2006, they used C++11. Although I really prefer the `make_unique` style, I will just add a C++11 compatible version using 'new'

Related tp #260

**Comment by davidscn** [comment]
If your OpenFOAM build uses cpp11, then we are bound to the OpenFOAM Cpp standard or we have to rebuild OpenFOAM on the cluster, which seems tedious.

**Comment by MakisH** [comment]
I think that this message is only due to the default options of OpenFOAM, which WMake uses. We should still be able to link a C++11 OpenFOAM to a C++14 adapter.

What I meant was specifying a different standard, e.g., by setting `-std=c++14`) as an additional compiler flag. But this is a discussion for another issue.

**Comment by davidscn** [comment]
This was closed as part of #264
